spec/OpenLineage.json

```diff
@@ -0,0 +1,426 @@
+{
+  "$schema": "http://json-schema.org/schema#",
```

Comment:
Should we add the [specific draft](https://json-schema.org/specification-links.html#published-drafts) of the standard we are using? For example: `http://json-schema.org/draft-07/schema#`

Reply:
good point